public/styles/main.css

```diff
@@ -1,18 +1,58 @@
+/* GENERICS - project wide ------------------------ 
+- TO Maybe: once enough generics - abstract to it's own file?
```

Comment:
yes we should have these in a separate file. Normally I'd call this a "reset" so maybe `reset.css`?

Reply:
This has been updated, just waiting on checks.